playbooks/02 - Use Case - Phishing Email Response/Investigate Suspicious Email.json

```diff
@@ -198,7 +198,7 @@
                 "input": {
                     "schema": {
                         "title": "Approval - Block Malicious Indicators",
-                        "description": "<p>Please approve the blocking of the unblocked indicators:<\/p>\n{% if vars.blocked_iocs | length > 0%}\n**Blocked Indicators:**\n{% for i in vars.blocked_iocs %}\n* {{i[0]}} - {{i[1]}}\n{%endfor%}\n{% endif %}\n**Unblocked Indicators:**\n{% for i in vars.unblocked_iocs %}\n* {{i[0]}} - {{i[1]}}\n{%endfor%}",
+                        "description": "{% if vars.blocked_iocs | length > 0%}\n###### **Already Blocked Malicious Indicators:**\n{% for i in vars.blocked_iocs %}\n* {{i[0]}} - {{i[1]}}\n{%endfor%}\n{% endif %}\n###### **Malicious Indicators Pending Blocking Approval:**\n{% for i in vars.unblocked_iocs %}\n* {{i[0]}} - {{i[1]}}\n{%endfor%}\n<p><span class=\"badge badge-pill badge-danger\" style=\"background: #ba6204; color: #fff; padding: 2px 7px;\"><strong>Action Required:<\/strong><\/span>&nbsp;Please review and approve the blocking of the listed indicators.<\/p>",
```

Comment:
Malicious Indicators - Already Blocked
Malicious Indicators - Pending Block Approval

Reply:
Button name could be approve instead of yes/no

Reply:
Done

<img width="689" alt="image" src="https://github.com/fortinet-fortisoar/solution-pack-phishing-email-response/assets/46894140/117e3273-b5d3-4cb9-81fa-85c04c31ae98">